app/javascript/components/search/FormFilter.js

```diff
@@ -0,0 +1,77 @@
+import mapKeys from "lodash/mapKeys";
+import React from "react";
+import PropTypes from "prop-types";
+import Button from "react-bootstrap/lib/Button";
+import Popover from "react-bootstrap/lib/Popover";
+import OverlayTrigger from "react-bootstrap/lib/OverlayTrigger";
+import Select2 from "react-select2-wrapper/lib/components/Select2.full";
+
+import "react-select2-wrapper/css/select2.css";
+
+/**
+ * Converts a list of forms from the backend into something Select2 understands.
+ */
+const parseFormsForSelect2 = (allForms) => allForms
+  .map((form) => mapKeys(form, (value, key) => key === "displayName" ? "text" : key));
```

Comment:
Why is this outside of the class? Seems like something that would be a private method. Is this how private methods are accomplished in React?

Reply:
This is effectively a private method (though JavaScript doesn't have that concept), but more importantly, it's static.

I could have instead declared it a `static` class method but that's messier to factor out. I like putting utils outside of the class for ease of future refactoring. As soon as some other component wants to use this logic, it can move into a shared `utils.js` or something.